### DISCUSSION TITLE
On Windows 10 Enterprise LTSC 2019, no Task Scheduler entry is created

### DISCUSSION THREAD

**Discussion by Terri-Kennedy** [kangyu-california/PersistentWindows#231] [Q&A]
When the program starts, no Task Scheduler entry is created, even after doing "Capture windows to disk". This is regardless of whether I do "Run as administrator" or not. Is the intention for each user to create the Task Scheduler entry themselves or is it something the program normally does? The README.md file just shows the Settings tab and not any of the other tabs that presumably need to be populated.

Is this something that has gone wrong due to using .NET 6.0 instead of 4.8 (see my preceding question) or is it something else?

If this is something users are expected to manually do after installing, a set of screenshots of each tab would be useful. A PowerShell script to install the task would be wonderful, although it might need editing to reflect the user's installation directory, username, etc.

**Comment by kangyu-california** [comment]
For the time being users need to manually create auto-startup task in task scheduler.
I asked your question to chatGPT, and here is the answer

The .ps script requires to run with admin privilege.

#### Change the path ####
$executablePath = "C:\Path\to\Your\PersistentWindows.exe"
####  rename the task as you like ####
$taskName = "StartupPersistentWindows"

$taskDescription = "This task starts automatically on ANY user login."

$action = New-ScheduledTaskAction -Execute $executablePath
$trigger = New-ScheduledTaskTrigger -AtLogOn

$settings = New-ScheduledTaskSettingsSet -AllowStartIfOnBatteries -DontStopIfGoingOnBatteries

Register-ScheduledTask -TaskName $taskName -Action $action -Trigger $trigger -Settings $settings -Description $taskDescription

$task = Get-ScheduledTask -TaskName $taskName
$taskSettings = $task.Settings
#### Sets the execution time limit to 3650 days ####
$taskSettings.ExecutionTimeLimit = "P3650DT0H0M0S"  
Set-ScheduledTask -TaskName $taskName -Settings $taskSettings

**Reply by Terri-Kennedy** [reply]
Thanks! Between this and some screenshots I found in other discussions, I think I have things set up properly. I'll be able to verify tomorrow (this is on a series of 80" display panels, so I can't just turn them off and back on during the day to see if I've got things configured correctly).
